integration-cli/docker_cli_by_digest_test.go

```diff
@@ -115,7 +115,13 @@ func testPullByDigestNoFallback(c *testing.T) {
 	imageReference := fmt.Sprintf("%s@sha256:ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff", repoName)
 	out, _, err := dockerCmdWithError("pull", imageReference)
 	assert.Assert(c, err != nil, "expected non-zero exit status and correct error message when pulling non-existing image")
-	assert.Assert(c, strings.Contains(out, fmt.Sprintf("manifest for %s not found", imageReference)), "expected non-zero exit status and correct error message when pulling non-existing image")
+
+	expectedMsg := fmt.Sprintf("manifest for %s not found", imageReference)
+	if testEnv.UsingSnapshotter() {
+		expectedMsg = fmt.Sprintf("%s: not found", imageReference)
+	}
```

Comment:
We should really ditch these tests and have a test in `integration/` (checking the error-response as well as status, instead of just the string-matching).